DESCRIPTION

```diff
@@ -6,14 +6,14 @@ Description: Provided with a Socrata dataset resource URL,
     returns an R data frame.
     Converts dates to POSIX format.
     Manages throttling by Socrata.
-Version: 1.5.1
-Date: 2014-12-08
+Version: 1.5.2
```

Comment:
Just pushed an updated version with the adjusted version number.

Tom Schenk Jr writes:

>> @@ -6,14 +6,14 @@ Description: Provided with a Socrata dataset resource URL,
>>      returns an R data frame.
>>      Converts dates to POSIX format.
>>      Manages throttling by Socrata.
>> -Version: 1.5.1
>> -Date: 2014-12-08
>> +Version: 1.5.2
>
> We're going accept this as a beta for version 1.6.0, which will be submitted to CRAN in a month or so. Can you adjust this to "1.6.0-beta"?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Chicago/RSocrata/pull/12/files#r22690710